x-pack/plugins/security_solution/public/management/components/artifact_list_page/hooks/use_with_artifact_list_data.ts

```diff
@@ -46,7 +48,14 @@ export const useWithArtifactListData = (
   const isMounted = useIsMounted();
 
   const {
-    urlParams: { page = 1, pageSize = MANAGEMENT_DEFAULT_PAGE_SIZE, filter, includedPolicies },
+    urlParams: {
+      page = 1,
+      pageSize = MANAGEMENT_DEFAULT_PAGE_SIZE,
+      filter,
+      includedPolicies,
+      sortField = MANAGEMENT_DEFAULT_SORT_FIELD,
+      sortOrder = MANAGEMENT_DEFAULT_SORT_ORDER,
```

Comment:
I don't think these two attributes should be persisted/retrieved from the URL, because the user does not have a way to sort columns on a list. They should be set internally only to our hard coded values.

I just noticed that the type `ArtifactListPageUrlParams` has these defined there - that's a mistake, I think and we should probably remove it.

Reply:
Yeah, I thought about that as well, but I wasn't sure if we should entirely ignore URL params added by the user for sorting. I'll go ahead and hard-code it for now.

Reply:
done e613c62ee2c64c0e8bddad4adb432fb82f912ace